webapp/layouts/error.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="error-container">
+    <img class="error-image" :src="image" />
+    <br />
+    <span class="error-message">{{ $t(message) }}</span>
+    <br />
+    <nuxt-link to="/">{{ $t('error-pages.back-to-index') }}</nuxt-link>
+  </div>
+</template>
+
+<script>
+export default {
+  props: ['error'],
+  computed: {
+    message() {
+      const mapping = {
+        403: 'error-pages.403-default',
+        404: 'error-pages.404-default',
+        500: 'error-pages.500-default',
+        503: 'error-pages.503-default',
+      }
+      return this.error.key || mapping[this.error.statusCode] || 'error-pages.default'
+    },
+    image() {
+      return `/img/svg/errors/error${this.error.statusCode || '500'}.svg`
+    },
+  },
+}
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-28T21:06:57Z" title="Tuesday, January 28th 2020, 10:06:57 pm +01:00">Jan 28, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="error-container">
+    <img class="error-image" :src="image" />
+    <br />
+    <span v-if="error.message === 'This page could not be found'" class="error-message">
+      {{ $t('error-pages.default') }}
+    </span>
+    <span v-else class="error-message">{{ $t(error.message) }}</span>
+    <br />
+    <nuxt-link to="/">{{ $t('error-pages.back-to-index') }}</nuxt-link>
+  </div>
+</template>
+
+<script>
+export default {
+  props: ['error'],
+  computed: {
+    message() {
+      const mapping = {
+        403: 'error-pages.403-default',
+        404: 'error-pages.404-default',
+        500: 'error-pages.500-default',
+        503: 'error-pages.503-default',
+      }
+      return this.error.key || mapping[this.error.statusCode] || 'error-pages.default'
+    },
+    image() {
+      return `/img/svg/errors/error${this.error.statusCode ? this.error.statusCode : '500'}.svg`
```


```suggestion
      return `/img/svg/errors/error${this.error.statusCode || '500'}.svg`
```

Reply:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-01-28T21:28:21Z" title="Tuesday, January 28th 2020, 10:28:21 pm +01:00">Jan 28, 2020</time>_

---


@roschaefer I thought you wanted an image :)